pyaerocom/helpers.py

```diff
@@ -1682,3 +1684,73 @@ def get_time_rng_constraint(start, stop):
     t_upper = iris.time.PartialDateTime(year=stop.year, month=stop.month, day=stop.day)
 
     return iris.Constraint(time=lambda cell: t_lower <= cell <= t_upper)
+
+
+def get_max_period_range(periods):
+    start = min([int(per.split("-")[0]) for per in periods])
```

Comment:
you can pass the generator expression directly to `min` without creating a list, e.g.

``` python
    start = min(int(per.split("-")[0]) for per in periods)
```